lib/linkable.rb

```diff
@@ -0,0 +1,8 @@
+module Linkable
```

Comment:
Yeah ruby/rails organization still baffles me, not always sure where to put things. Just tried this and couldn't make the autoload part work.